apps/E2E/src/TabList/pages/TabListPageObject.ts

```diff
@@ -0,0 +1,77 @@
+import { BasePage, By } from '../../common/BasePage';
+import { Attribute, AttributeValue } from '../../common/consts';
+import {
+  TABLIST_TESTPAGE,
+  HOMEPAGE_TABLIST_BUTTON,
+  TABLIST_TEST_COMPONENT,
+  FIRST_TAB,
+  SECOND_TAB,
+  THIRD_TAB,
+  FOURTH_TAB,
+  FIFTH_TAB,
+  TABLIST_CALLBACK_TEXT,
+} from '../consts';
+
+type Tab = 'First' | 'Second' | 'Third' | 'Fourth' | 'Fifth';
+
+class TabListPageObject extends BasePage {
+  async resetListSelection(): Promise<void> {
+    return (await this.getTab('First')).click();
+  }
+
+  async isTabSelected(selector: Tab): Promise<boolean> {
+    return await (await this.getTab(selector)).isSelected();
+  }
+
+  async isTabFocused(selector: Tab): Promise<boolean> {
+    return await this.compareAttribute(this.getTab(selector), Attribute.IsFocused, AttributeValue.true);
+  }
+
+  async waitForTabSelected(selector: Tab, errorMsg: string, timeout?: number): Promise<boolean> {
```

Comment:
Instead of returning `return await this.isTabSelected(selector);`, we can do:
`return await this.waitForCondition(async () => await this.isTabSelected(selector), errorMsg, timeout);`

This improves perf by not having unnecessary calls

Reply:
`waitForCondition` returns void, but it does throw an error if the condition isn't met. I removed the return anyway for one less await call.